components/contact/ContactSectionRow.tsx

```diff
@@ -1,9 +1,29 @@
 import { Fragment } from 'react'
 import Markdown from 'markdown-to-jsx'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+// import { IconDefinition } from '@fortawesome/fontawesome-common-types';
 import { icon } from '../../lib/loadIcons'
 
-function ContactSectionRow(props) {
+interface DetailItem {
+  link: string
+  content: string
+}
+
+interface ContactSectionRowProps {
+  label?: string
+  detail?: string
+  index: number
+  highlight?: boolean
+  iconFeature?: string
+  button?: boolean
+  buttonURL?: string
+  buttonId?: string
+  items: DetailItem[]
+  refPageAA: string
+  id: string
+}
+
+function ContactSectionRow(props: ContactSectionRowProps) {
```

Comment:
Let's keep this consistent with the rest of the components and do:
```suggestion
const ContactSectionRow = ({
  label,
  detail,
  index,
  highlight,
  iconFeature,
  button,
  buttonId,
  items,
  refPageAA,
  id,
}: ContactSectionRowProps) => {
```
We can then remove the initialization on line 38-39